pkg/app/var.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-sigs/kustomize/pkg/resource"
+	"github.com/kubernetes-sigs/kustomize/pkg/types"
+)
+
+func gvkn(rv types.Var) resource.ResId {
+	return resource.NewResId(rv.ObjRef.GroupVersionKind(), rv.ObjRef.Name)
+}
+
+func getFieldAsString(m map[string]interface{}, pathToField []string) (string, error) {
```

Comment:
unit test for this please, mainly to document usage (even though its private).
its a little too complex to not have coverage.

Reply:
I will add unit test for it.